src/Dotnet.Installer.Console/Commands/ListCommand.cs

```diff
@@ -47,14 +58,41 @@ private async Task Handle()
                     components[installedComponent.Name] = (Installed: true, Version: version);
                 }
 
-                var dotNetRuntimeStatus = ComponentStatus(Constants.DotnetRuntimeComponentName, majorVersionGroup.Key);
-                var aspNetCoreRuntimeStatus = ComponentStatus(Constants.AspnetCoreRuntimeComponentName, majorVersionGroup.Key);
-                var sdkStatus = ComponentStatus(Constants.SdkComponentName, majorVersionGroup.Key);
+                var endOfLife = majorVersionGroup.First().EndOfLife;
+                var isEndOfLife = endOfLife < DateTime.Now;
+
+                var dotnetVersionStringBuilder = new StringBuilder();
+                dotnetVersionStringBuilder.Append($".NET {majorVersionGroup.Key}");
+                if (majorVersionGroup.First().IsLts)
+                    dotnetVersionStringBuilder.Append(" LTS");
+
+                var dotNetRuntimeStatus = ComponentStatus(
+                    Constants.DotnetRuntimeComponentName,
+                    majorVersionGroup.Key,
+                    isEndOfLife);
+                var aspNetCoreRuntimeStatus = ComponentStatus(
+                    Constants.AspnetCoreRuntimeComponentName,
+                    majorVersionGroup.Key,
+                    isEndOfLife);
+                var sdkStatus = ComponentStatus(Constants.SdkComponentName,
+                    majorVersionGroup.Key,
+                    isEndOfLife);
+
+                var eolStringBuilder = new StringBuilder();
+                eolStringBuilder.Append(isEndOfLife ? "[bold red]" : "[bold green]");
+                eolStringBuilder.Append(endOfLife.ToShortDateString());
+                eolStringBuilder.Append("[/]");
+
+                table.AddRow(
+                    dotnetVersionStringBuilder.ToString(),
+                    dotNetRuntimeStatus,
+                    aspNetCoreRuntimeStatus,
+                    sdkStatus,
+                    eolStringBuilder.ToString());
```

Comment:
Yeah, I usually prefer `StringBuilder` to avoid the extra string allocations, but I guess that, at such a small scale, this should be fine and easier to read.

Reply:
It's just a suggestion. `StringBuilder` also doesn't cost a lot of performance. If you prefer `StringBuilder`, than that is fine too. Readability/consistency is more important than micro performance gains.